docs/help-center/using-famedly/messages.mdx

```diff
@@ -280,6 +280,35 @@ The message will be reported to the admin of your Organization.
   </TabItem>
 </Tabs>
 
+## Copy messages
+
+<Tabs>
+  <TabItem value="desktop" label="Desktop" default>
+  <ul>
+    <li>Select and copy text fragments from messages:</li>
+    <ol>
+    <li>Place the cursor at the start of the text you want to select.</li>
+    <li>Click and hold the left mouse key, then quickly move the cursor to the end of the text you want to select.</li>
+    <li>Press Ctrl+C to copy the text.</li>
```

Comment:
Or Command+C on Apple - how have we distinguished that before? Do we assume Windows?